privly.safariextension/scripts/background_scripts/mirror_local_storage.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @fileOverview This file mirrors the localStorage of the
+ * PRIVLY_APPLICATION pages in the background page. Whenever,
+ * the localStorage is updated in the PRIVLY_APPLICATION pages,
+ * a message is sent to the background page. This file checks
+ * for the updates in the localStorage and updates the background
+ * page localStorage.
+ */
+
+/**
+ * Make appropriate updates to localStorage of background page.
+ *
+ * @param event the message that is received
+ *
+ */
+safari.application.addEventListener("message", function(event) {
```

Comment:
For unit testing and documentation purposes, it is better to separate the listener function from the addEventListener.